ecowitt2mqtt/data.py

```diff
@@ -50,6 +51,8 @@
     DATA_POINT_SOLARRADIATION_LUX: calculate_illuminance_wm2_to_lux,
     DATA_POINT_SOLARRADIATION_PERCEIVED: calculate_illuminance_wm2_to_perceived,
     DATA_POINT_WINDCHILL: calculate_wind_chill,
+    # Prevent WINDDIR being converted by GLOB_WIND
```

Comment:
```suggestion
    # Prevent WINDDIR being converted by GLOB_WIND:
```

Reply:
Inline comments use `:` as their final character in this library; let's ensure we're consistent.